src/app/base/components/dropdown/dropdown.component.html

```diff
@@ -40,13 +40,14 @@
     </a>
 
     <div #dropdownOptionsDiv
-        class="origin-top-right absolute mt-2 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none hidden"
+        class="origin-top-right mt-2 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none hidden"
```

Comment:
I don't know that much about css classes, but the last class used here is hidden -> does this hide the whole element? Wouldn't the other classes then be unnecessary? Most probably the hidden does something different, but since I am a total noob in css I am interested in what this does :D

Reply:
I removed the 'absolute' class from the dropdown because I added it as an optional parameter to the dropdown options:
`[style.position]="dropdownOptions.useRelativePosOptions ? 'relative' : 'absolute'"`
So, again it's default as absolute but if the user needs it for some specific dropdown as relative, he can use it by sending the field of the options as true (I needed it for one dropdown :D )

Reply:
Ah okay thanks for explaining :)